File: week03/week03-1.cpp
//Week03-1.cpp 二合一的 C++ 的版本
//LeetCode 2579. Count Total Number of Colored Cells
class Solution {
public:
    long long coloredCells(int n) {
      long long ans = n; // 要多寫這行, 把數字升級, 變64位元「很長很長的整數」
      return ans*ans + (ans-1)*(ans-1); //C/C++/Java/C#都要加分號


    }
};

File: week03/week03-1.py
#Week03-1.py 二合一 Python 的版本
#LeetCode 2579. Count Total Number of Colored Cells
class Solution:
    def coloredCells(self, n: int) -> int:
        return n*n + (n-1)*(n-1)
        #觀察題目的圖片(不看英文、不看中文), 頭轉45度, 看到正方形

